README.md

```diff
@@ -216,7 +217,7 @@ NOTE: The db info used here will need to be used in the config as detailed in th
 
 ## Connecting your own opensearch cluster
 
-You will need to specify `OPEN_SEARCH_HOST`, `OPEN_SEARCH_USERNAME`, `OPEN_SEARCH_PASSWORD` `OPEN_SEARCH_CA_CERTS`, the middle 2 can be left empty if running with the AWS config as that would use AWS4Auth signing and the final one can be left empty if running on a system where the certificate for the opensearch cluster is already trusted by the system.
+You will need to specify `OPEN_SEARCH_HOST`, `OPEN_SEARCH_USERNAME`, `OPEN_SEARCH_PASSWORD` `OPEN_SEARCH_CA_CERTS`, `OPEN_SEARCH_TIMEOUT` the middle 2 can be left empty if running with the AWS config as that would use AWS4Auth signing and the third one can be left empty if running on a system where the certificate for the opensearch cluster is already trusted by the system.
```

Comment:
Might be good to rephrase this as "middle 2" can be slightly ambiguous to others with 5 ENV VARS now listed. (I know its OPEN_SEARCH_USERNAME & OPEN_SEARCH_PASSWORD).

```suggestion
You will need to specify the following ENV VARIABLES:

`OPEN_SEARCH_HOST` (Required) : You need to specify the OpenSearch cluster's host (URL).
`OPEN_SEARCH_USERNAME` and `OPEN_SEARCH_PASSWORD` (Optional): These are for basic authentication. If you're using AWS (with AWS4Auth signing), you can leave these empty since AWS will handle authentication.
`OPEN_SEARCH_CA_CERTS` (Optional): This is for specifying the CA (Certificate Authority) certificates used to verify SSL connections. You can leave this empty if running on a system where the certificate for the OpenSearch cluster is already trusted by the system.
`OPEN_SEARCH_TIMEOUT`: This defines the timeout duration for connections to OpenSearch.

```